### PR TITLE
Remove O-Phase specific stuff

### PR DESCRIPTION
**Pull request by pavelzw**

### CODE REVIEW

File: templates/index.html [hunk 1]
```diff
@@ -73,12 +73,19 @@ <h2>
...
+						<a class="button" href="fahrplan"><span class="icon fa-calendar"></span> Fahrplan</a>
+						-->
 						<a class="button" href="https://entenjagd.kitmatheinfo.de/"><span class="icon fa-keyboard"></span> CTF Minigame</a>
 					</p>
+					<!-- O-Phase Calendar iframe -->
+					<!--
 					<iframe src="https://calendar.google.com/calendar/embed?height=600&wkst=2&bgcolor=%23ffffff&ctz=Europe%2FAmsterdam&showCalendars=0&showNav=0&showTitle=1&showDate=0&mode=WEEK&src=OTAzZTE2MmRlODFhNWZiZDFhMTc3Nzc2MGY4ODU4ZmY2YzI1NTUzOThiOGExNjQ5ZjdiYzg2Y2NkNmUyODQ1YUBncm91cC5jYWxlbmRhci5nb29nbGUuY29t&color=%23009688&dates=20221017/20221018" style="border:solid 1px #777" width="800" height="600" frameborder="0" scrolling="no"></iframe>
 					<p>Mit einem Stern* markierte Termine gehören zum Programm der Fachschaft und werden nicht von uns geplant oder durchgeführt.</p>
 					<p>Wenn du nicht weißt wo wir sind, ruf die Nummer auf deiner Eintrittskarte an oder frag im #o-phase Channel. Wie du in den Channel kommst, steht auf der Tafel im Seminarraum.</p>
+					-->
```
Comment (libaurea) [inline templates/index.html:88]: Why not delete directly instead of commenting out? It is still in the git history at the latest.

**Review by libaurea — CHANGES_REQUESTED** [review]

File: templates/index.html (hunk 1 above)
Comment (pavelzw) [reply templates/index.html:88]: If we delete everything, we may forget to add the button to `kitmatheinfo.de/fahrplan` again 🤔

Comment (libaurea) [reply templates/index.html:88]: fair enough

**Review by libaurea — APPROVED** [review]
LGTM

**Pull request merged**